inst/qml/rareEventCharts.qml

```diff
@@ -0,0 +1,392 @@
+			singleVariable:						true
+		}
+
+		AssignedVariablesList
+		{
+			name:								"stage"
+			title:								qsTr("Stage")
+			id:									stages
+			allowedColumns:						["nominal"]
+			singleVariable:						true
+		}
+	}
+
+	RadioButtonGroup
+	{
+		name:								"dataType"
+		title: 								qsTr("Data type")
+		id:									dataType
+
+		RadioButton
+		{
+			value: 							"dataTypeDates"
+			label: 							qsTr("Date/time")
+			checked:		 				true
+
+			DropDown
+			{
+				name: 					"dataTypeDatesStructure"
+				id: 					dataTypeDatesStructure
+				label: 					qsTr("Structure")
+				values: 
+				[
+					{ label: qsTr("Date + Time"),						value: "dateTime"},
+					{ label: qsTr("Time + Date"),						value: "timeDate"},
+					{ label: qsTr("Date only"),							value: "dateOnly"},
+					{ label: qsTr("Time only"),							value: "timeOnly"}				
+				]
+				indexDefaultValue: 0
+			}
+
+			DropDown
+			{
+				name: 					"dataTypeDatesFormatDate"
+				id: 					dataTypeDatesFormatDate
+				label: 					qsTr("Date format")
+				visible:				dataTypeDatesStructure.value != "timeOnly"
+				values: 
+				[
+					{ label: qsTr("DMY"),							value: "dmy"},
+					{ label: qsTr("MDY"),							value: "mdy"},
+					{ label: qsTr("YMD"),							value: "ymd"},
+					{ label: qsTr("DM"),							value: "dm"},
+					{ label: qsTr("MD"),							value: "md"}
+				]
+				indexDefaultValue: 3
+			}
+
+			DropDown
+			{
+				name: 					"dataTypeDatesFormatTime"
+				id: 					dataTypeDatesFormatTime
+				label: 					qsTr("Time format")
+				visible:				dataTypeDatesStructure.value != "dateOnly"
+				values: 
+				[
+					{ label: qsTr("H"),								value: "H"},
+					{ label: qsTr("HM"),							value: "HM"},
+					{ label: qsTr("HMS"),							value: "HMS"},
+					{ label: qsTr("Ip"),							value: "Ip"},
+					{ label: qsTr("IMp"),							value: "IMp"}				
+				]
+				indexDefaultValue: 1
+			}
+		}
+
+		RadioButton
+		{
+			value: 							"dataTypeInterval"
+			label: 							qsTr("Interval between events")
+			childrenOnSameRow:				false
+
+			DropDown
+			{
+				name: 					"dataTypeIntervalType"
+				id: 					dataTypeIntervalType
+				label: 					qsTr("Interval type")
+				values: 
+				[
+					{ label: qsTr("Opportunities"),								value: "opportunities"},
+					{ label: qsTr("Time"),										value: "time"},
+					{ label: qsTr("Hours (decimal)"),							value: "hours"},
+					{ label: qsTr("Days (decimal)"),							value: "days"}				
+				]
+				indexDefaultValue: 0
+			}
+
+			DropDown
+			{
+				name: 					"dataTypeIntervalTimeFormat"
+				id: 					dataTypeIntervalTimeFormat
+				label: 					qsTr("Time format")
+				visible:				dataTypeIntervalType.value == "time"
+				values: 
+				[
+					{ label: qsTr("H"),								value: "H"},
+					{ label: qsTr("HM"),							value: "HM"},
+					{ label: qsTr("HMS"),							value: "HMS"},
+					{ label: qsTr("Ip"),							value: "Ip"},
+					{ label: qsTr("IMp"),							value: "IMp"}				
+				]
+				indexDefaultValue: 1
+			}
+		}
+
+	}
```

Comment:
@JorisGoosen don't you suddenly get an urge to create a date/ datetime column when you see this? :stuck_out_tongue: